src/components/Retool.js

```diff
@@ -1,9 +1,65 @@
 import { useEffect, useRef, useState } from "react";
 
-const Retool = ({ data, url, height, width, onData }) => {
+const IFramePermissionKeys = set(['downloads', 'forms', 'modals', 'popups', 'fullscreen', 'camera', 'microphone', 'geolocation', 'payment', 'topNavigation', 'sameOrigin'])
+
+const constructPermissionStrings = (permissions) => {
+  if (!permissions || permissions.length === 0) {
+    return { allowString: undefined, sandboxString: undefined }
+  }
+
+  const allowArray = []
+  const sandboxArray = []
```

Comment:
I think if Retool is sandboxed it will always require `allow-scripts` and `allow-same-origin`. Should we just add them by default?

Reply:
Oh good call - yeah I think those are reasonable to have